### ISSUE TITLE
Progress report for async execution?

### ISSUE THREAD

**Issue by expez** [Malabarba/paradox#93]
It would be great if the report buffer was updated as paradox did it's thing instead of once everything is complete.  About 5 minutes ago I initiated an upgrade of 15 packages and the spinner is still going. I've no idea about the status of paradox.  Most likely it just failed silently.

**Comment by Malabarba** [comment]
Yep, it would be a nice feature, and it's been requested before.
I just haven't done it cause it's a lot of work. Implementing it will require us parsing the output emited by the background emacs process, which is usually filled with all kinds of garbage. :P
